components/album/AlbumItem.vue

```diff
@@ -7,7 +7,7 @@ defineProps<{
 </script>
 
 <template>
-  <li class="group relative mr-3 cursor-pointer py-3">
+  <li class="group col-span-5 relative mr-3 cursor-pointer py-3">
```

Comment:
Could we do `col-span-full` here? So we don't have to change this manually if we decide to add another column later